addons/web/static/src/legacy/scss/fields.scss

```diff
@@ -542,6 +542,9 @@
         font-weight: 500;
         @include o-text-overflow;
         transition: none; // remove transition to prevent badges from flickering at reload
+        &.o_field_empty {
+            display: none !important;
```

Comment:
is `!important` really necessary?

Reply:
It was prior to the owl / css refactoring. I'll test it again. 

But at the moment I wrote this, it was :)